sdk/mediaservices/Microsoft.Azure.Media.LiveVideoAnalytics.Edge/src/Microsoft.Azure.Media.LiveVideoAnalytics.Edge.csproj

```diff
@@ -2,12 +2,12 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Media.LiveVideoAnalytics.Edge</PackageId>
     <Description>Provides developers with libraries for managing Live video analytics edge module using direct methods</Description>
-    <Version>1.0.0-preview.2</Version>
```

Comment:
Why not move to `1.0.0-preview.3` ?

Reply:
trying to keep the version same as the actual LVA module user will install on the edge device. so that its easier for them to correlate the SDK version with the module.

Reply:
Ok.